compute/standard-deviation/docs/modules/ROOT/pages/compute-standard-deviation.adoc

```diff
@@ -0,0 +1,332 @@
+:github-directory: https://github.com/hazelcast/hazelcast-cloud-code-samples/tree/master/compute/standard-deviation
+
+{description}
+
+== Context
+
+Knowing the center of a dataset (average, mean, or median) such as customer satisfaction scores doesn't provide enough insight into the best and worse scores. To find out how the customer satisfaction scores are spread out, you can use link:https://en.wikipedia.org/wiki/Standard_deviation[standard deviation].
+
+Standard deviation is a statistical calculation that has been adopted for calculating how numbers deviate from the average. The business logic for standard deviation includes these steps:
+
+. Calculate the average of the input numbers.
+. Calculate the running total of the square of how much each number differs from the average.
+. Calculate the average of the running total.
+. Take the square root of the running total divided by the number of inputs and call it the standard deviation.
+
+The second step is open to *parallelism*. If you have more than one CPU, you can run this step for several numbers at once, as long as you have a way to combine these independent answers into the running total. *This is what you'll use Hazelcast for*.
+
+Here is an example of standard deviation for two sets of numbers.
+
+.First set of numbers
+[cols="4"]
+|===
+|Number |Difference from the average (3)|Square| Running total
+
+|1
+|2
+|4
+|4
+
+|2
+|1
+|1
+|5
+
+|3
+|0
+|0
+|5
+
+|4
+|1
+|1
+|6
+
+|5
+|2
+|4
+|10
+
+|===
+
+.Second set of numbers
+[cols="4"]
+|===
+|Number |Difference from the average (3)|Square| Running total
+
+|1
+|2
+|4
+|4
+
+|1
+|1
+|4
+|8
+
+|3
+|0
+|0
+|8
+
+|5
+|2
+|4
+|12
+
+|5
+|2
+|4
+|16
+
+|===
+
+The running total for the first set is 10 and there were 5 numbers. 10 divided by 5 is 2. The square root of 2 gives the standard deviation of *1.41*.
+
+The running total for the second set is 16 and there were 5 numbers. 16 divided by 5 is 3.2. The square root of 3.2 gives the standard deviation of *1.78*.
+
+This proves that the first set of numbers doesn't vary as much as the second set.
+
+In this tutorial, you'll deploy the following to {hazelcast-cloud}:
+
+- A `Customer` class that you'll use to store `Customer` objects in a map. The objects will include a customer satisfaction score.
+- An EntryProcessor that the cluster will use to calculate the standard deviation of all customer satisfaction scores in the map.
+
+You'll then use a client to connect to the cluster, load some `Customer` objects into a map, and calculate the standard deviation of all customer satisfaction scores.
+
+The client code shows both the junior developer and senior developer approach. The junior developer approach does the processing on the client-side. The senior developer approach is more efficient because it offloads the processing to {hazelcast-cloud}, using the EntryProcessor.
```

Comment:
```suggestion
The client code shows both the junior developer and senior developer approach. The junior developer approach does the processing on the client-side. The senior developer approach is more efficient because it offloads the processing to {hazelcast-cloud} using the entry processor.
```